### ISSUE TITLE
Multiple widgets used the same GlobalKey.

### ISSUE THREAD

**Issue by mlican** [daohoangson/flutter_widget_from_html#1043]
```Dart
    String html = r'''
    <p class="footnote"><sup><a id="fn323"></a ><a class="nounder" href=" ">5</a ></sup> The logic of time and tense was first studied systematically by A. N. Prior in <em>Time and Modality</em> (Oxford: Oxford University Press, 1957) and deontic logic by G. H. von Wright in <em>An Essay on Deontic Logic</em> (Amsterdam: North-Holland, 1968).</p >
    ''';
    return HtmlWidget(html);
```
The key [GlobalKey#c9e60 fn323] was used by multiple widgets. The parents of those widgets were different widgets that both had the following description:
  WidgetPlaceholder<String>(#fn323, dependencies: [TshWidget])
A GlobalKey can only be specified on one widget at a time in the widget tree.

**Comment by daohoangson** [comment]
This appears to be a bug between vertical align (the `SUP` tag) and anchor (the `A` tag). Let me investigate further and get back to you.

**Comment by daohoangson** [comment]
The bug is confirmed in the latest version (v0.10.5) but it is fixed in the latest alpha (v0.13.0). This version has a new layout algorithm so it handles corner cases better. Can you try the new version with your app?

**Comment by mlican** [comment]
> The bug is confirmed in the latest version (v0.10.5) but it is fixed in the latest alpha (v0.13.0). This version has a new layout algorithm so it handles corner cases better. Can you try the new version with your app?

I appreciate your prompt response. I will try the new version with my app.
Thanks again

**Comment by daohoangson** [comment]
The latest stable `v0.14.3` seem to handle your HTML without issue.

<img width="435" alt="Screenshot 2023-10-18 at 10 30 12" src="https://github.com/daohoangson/flutter_widget_from_html/assets/239336/c6865e1e-5ff0-44df-ad37-43ca1171895c">
